cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -236,6 +248,123 @@ func canInstallAndRunAtOnce(mst *initramfsMountsState) (bool, error) {
 	return kernelHasFdeSetup, nil
 }
 
+func getSnapInfo(sysSnaps map[snap.Type]*seed.Snap, snapType snap.Type) (*snap.Info, error) {
```

Comment:
s/get/read would be more consistent naming

Reply:
Done